hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/functional/TestCleanPlanExecutor.java

```diff
@@ -263,11 +264,9 @@ public void testKeepLatestCommits(
     assertTrue(testTable.baseFileExists(p0, "00000000000007", file4P0C3));
   }
 
-  /**
-   * Test Hudi COW Table Cleaner - Keep the latest file versions policy.
-   */
-  @Test
-  public void testKeepLatestFileVersions() throws Exception {
+  @ParameterizedTest
+  @ValueSource(Boolean.class)
+  public void testKeepLatestFileVersions(boolean ) throws Exception {
```

Comment:
missing an argument here? did you intend to test with and without incremental mode enabled?